### PR TITLE
Add one line for ranches. Add TMT example

### PR DESCRIPTION
**Pull request by phracek**
This commit adds one line to a better reading option for ranches.

Adds also TMT examples and fixes the links to README.md
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
[test]

File: USAGE.md [hunk 1]
```diff
@@ -31,6 +32,50 @@ How to test your TMT plans locally? Run a command:
...
+prepare:
+    - name: Install important packages for your tests
+      how: shell
+      script: |
+        # install packages on guest system
+
+    - name: Install packages on machine by ansible
+      how: ansible
+      playbook:
+        - ansible-plan.yml
```
Comment (zmiklank) [inline USAGE.md:61]: Where is this defined/explained?

Comment (phracek) [reply USAGE.md:61]: Good point. This is not defined anywhere this is an example. 

The question is if we want to show examples as ansible.

Comment (phracek) [reply USAGE.md]: Fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/68/commits/c39701a3ec39918bc76c4bddbedcb14eb7e05674

**Review by zmiklank — APPROVED** [review]

**Pull request merged**